templates/default/sentry.conf.8.0.py.erb

```diff
@@ -0,0 +1,205 @@
+# This file is just Python, with a touch of Django which means
```

Comment:
How different is this file from `sentry.conf.py.erb`?
Additive config keys should be fine.
And if any config keys have different values between v7 and v8, you can make the appropriate changes before rendering the template.